src/annotator/annotation-sync.coffee

```diff
@@ -21,13 +21,18 @@ module.exports = class AnnotationSync
     @_on = options.on
     @_emit = options.emit
 
+    iframe = document.createElement('iframe')
+    iframe.style.display = 'none'
+    document.body.appendChild(iframe)
+    bind = iframe.contentWindow.Function.prototype.bind
```

Comment:
I think this would make sense. It seems likely that we might want to do this in other places in future. It should also be much easier to test if you extract it out into a separate module.